src/libraries/System.Net.Quic/src/Resources/Strings.resx

```diff
@@ -204,6 +204,9 @@
   <data name="net_quic_protocol_error" xml:space="preserve">
     <value>A QUIC protocol error was encountered</value>
   </data>
+  <data name="net_quic_alpn_in_use" xml:space="preserve">
+    <value>Another QUIC listener is already listening on one of the requested application protocols on the same port.</value>
```

Comment:
I am a bit confused about the wording of the error. If we ask for multiple ALPNs and one is in use but the second is available -- do we still fail?

Reply:
yes, the two ALPN lists must be fully disjoint.